components/badge/badge.css

```diff
@@ -5,6 +5,52 @@
   width: fit-content;
   box-sizing: border-box;
   margin: 0;
+  color: var(--black);
+}
+
+.avatar-badge {
+  display: flex;
+  position: relative;
+  align-items: center;
+  gap: 10px;
+  width: fit-content;
+  box-sizing: border-box;
+  margin: 0;
+  color: var(--black);
+}
+.avatar-badge .badge {
+  position: absolute;
+  right: -0.2rem;
+  bottom: 0.2rem;
+  z-index: 1;
+  width: 0.8rem;
+  height: 1.5rem;
+  border: 2px solid var(--white);
+  border-radius: 50%;
+}
+
+.icon-badge {
+  display: flex;
+  position: relative;
+  align-items: center;
+  gap: 10px;
+  width: fit-content;
+  box-sizing: border-box;
+  margin: 0;
+  color: var(--black);
+}
+.icon-badge i {
+  font-size: 2.5rem;
+}
+.icon-badge .badge {
```

Comment:
Most of the properties for `.avatar-badge .badge ` and `.icon-badge .badge`  are same. You can merge the common properties and only separate the ones that are different